src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -143,5 +143,6 @@ def _set_user_status_on_project(project, user, status, role, enable_notification
                 role=ProjectUserRoleChoice.objects.get(name=role),
                 enable_notifications = enable_notifications
             )
+            pu.save()
```

Comment:
This is the wrong place for the save. It should be on update, after line 137, not on create.